packages/components/src/components/dropdown/dropdown.js

```diff
@@ -239,7 +239,9 @@ class Dropdown extends mixin(
     );
     const start =
       this.getCurrentNavigation() ||
-      this.element.querySelector(this.options.selectorItemSelected);
+      this.element.querySelector(
+        `${this.options.selectorItemSelected} ${this.options.selectorItem}`
```

Comment:
We don't do string interpolation for selectors in code. In this kind of case, we introduce a new option, like `selectorLinkSelected`.